src/components/ActivityMap/ActivityStats.tsx

```diff
@@ -86,6 +110,20 @@ export function ActivityStats({
                 units="min/mi"
               />
             </Grid>
+            <Grid>
+              <Statistic
+                name="🐶 Max Acceleration"
+                value={computeMaxAccel(garminActivity.records).toFixed(1)}
```

Comment:
Any noticable perf issues running this in the render function? Wonder if this is expensive enough to memoize

Reply:
I tested this, it's plenty fast